EMS/core-bundle/src/Resources/views/user/permissions/permissions.html.twig

```diff
@@ -0,0 +1,138 @@
+                            </thead>
+                            <tbody>
+                            {% for role in roles %}
+                                <tr>
+                                    <td>{{ loop.index }}</td>
+                                    <td>{{ role|trans }}</td>
+                                    <td>{{ roleDescriptions[role]|trans }}</td>
+                                </tr>
+                            {% endfor %}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <div class="box ">
+                <div class="box-header with-border">
+                    <i class="fa fa-group"></i>
+                    <h3 class="box-title">{{ 'view.user.permissions.titles.roles_specific_functionalities'|trans }}</h3>
+                </div>
+                <div class="box-body">
+                    <div class="table-responsive">
+                        <p>{{ 'view.user.permissions.titles.descriptions.roles_functionalities_description'|trans }}</p>
+                        <table class="table table-condensed table-striped">
+                            <thead>
+                            <tr>
+                                <th>{{ 'view.user.permissions.table_headers.role'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.description'|trans }}</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {% for roleData in [
+                                {'role': 'ROLE_API', 'description': 'view.user.permissions.roles_functionality.role_api'|trans},
+                                {'role': 'ROLE_FORM_CRM', 'description': 'view.user.permissions.roles_functionality.role_form_crm'|trans},
+                                {'role': 'ROLE_TASK_MANAGER', 'description': 'view.user.permissions.roles_functionality.role_task_manager'|trans},
+                                {'role': 'ROLE_ALLOW_ALIGN', 'description': 'view.user.permissions.roles_functionality.role_allow_align'|trans},
+                                {'role': 'ROLE_USER_MANAGEMENT', 'description': 'view.user.permissions.roles_functionality.role_user_management'|trans},
+                                {'role': 'ROLE_COPY_PASTE', 'description': 'view.user.permissions.roles_functionality.role_copy_paste'|trans},
+                                {'role': 'ROLE_DEFAULT_SEARCH', 'description': 'view.user.permissions.roles_functionality.role_default_search'|trans},
+                                {'role': 'ROLE_SUPER_USER', 'description': 'view.user.permissions.roles_functionality.role_super_user'|trans},
+                                {'role': 'ROLE_USER_READ', 'description': 'view.user.permissions.roles_functionality.role_user_read'|trans}
```

Comment:
this table can generated  from a simple array like ['ROLE_API','ROLE_FORM_CRM',....]